subworkflows/nf-core/fgbio_create_umi_consensus/main.nf

```diff
@@ -0,0 +1,86 @@
+//
+// Runs FGBIO tools to remove UMI tags from FASTQ reads
+// Convert them to unmapped BAM file, map them to the reference genome,
+// use the mapped information to group UMIs and generate consensus reads
+//
+
+params.groupumistrategy        = "Adjacency"
+params.aligner                 = "bwa-mem"
```

Comment:
Should that be in the config?

Reply:
more than that, they should have been passed as options with the previous syntax and now as ext.something with the new one. 
but this requires modifying the modules, and I didn't want to break anyone's code. 
if you advise for that, I'd be happy to fix this too and have everything clean and consistent.

Reply:
I'd advise for that because, for me a subworkflow should be like any module, without any hard-coded params.

Reply:
done and successfully tested (both subworkflow and updated module) :)